content/en/guides/employee-engagement.md

```diff
@@ -0,0 +1,179 @@
+* Have a look at closed issues and pull requests to learn how contributions work
+* Look for recordings of presentations of the project at conferences to learn more about the project and the people who speak about it
+
+
+## Principle 4: Has Transparent Motivations.
+
+*Without a shared understanding of the motivations, it’s impossible to resolve differences of opinion effectively. No hidden motives.*
+
+Your work on the projects should be transparent. Transparency has many different aspects an dimensions that we will try to showcase here.
+
+You should be transparent with the rest on the community about your level of commitment with this project: from working only on issues that overlap with your company's interests, to help out in anything the project needs, or anything in between; from doing a one-off fix to working towards long-term maintainership of some code. All the options are valid and there is not right or wrong answer.
+
+It is important to be transparent about the reason why you are working on the tasks you pick. Sharing as much context as you can is useful for the rest of the community to better gauge the importance of the work and why it should be included in the project. This is not about revealing company strategies, but sharing and explaining why is this particular piece of work needed and interesting.
+
+We have multiple hats when interacting with communities: our company's or our own. It is important to differentiate and be explicit about which virtual persona you are in each moment. For example, when committing changes or writing emails, use the corporate or the personal email accordingly ([more info](https://www.juliaferraioli.com/blog/2022/your-git-email-matters/)).
+
+You also most probably can't speak officially for your organization, but some people will still perceive you as speaker of your organization. Handle that with care. Don't play political games, this will fail.
+
+Some companies will have the expectation that they can steer an open source project, because they have money or status or customers. Some companies will try to impose a hidden agenda on a project. This is not how open source works. The currency is active contribution for the benefit of the project. It's ok to influence a project, and everybody will understand that a company is not contributing for pure idealistic motives. But this is done by sending individuals to the project who are doing valuable work. As an employee you are one of these individuals. So you will need to balance this and be transparent about your motivations.
+
+### Suggested tactics for having transparent motivations
+
+* Commit with your organization's email address when you are working on behalf of your organization
+* Put your affiliation on GitHub
+* Make it explicit in communication from which perspective you are speaking when recipients might not have this context
```

Comment:
* Propose changes and show impact. It might not succeed on your first try. Iterate. Keep learning.
* Don't be angry if you are ignored or a suggestion is rejected. This can happen. Motivations differ.
* Don't make demands. This will burn up your credibility.

Reply:
Incorporated in #397.